proxy/proxy.go

```diff
@@ -915,6 +915,9 @@ func (p *Proxy) makeBackendRequest(ctx *context, requestContext stdlibcontext.Co
 		return nil, &proxyError{err: fmt.Errorf("unexpected error from Go stdlib net/http package during roundtrip: %w", err)}
 	}
 	p.tracing.setTag(ctx.proxySpan, HTTPStatusCodeTag, uint16(response.StatusCode))
+	if response.Uncompressed {
+		p.metrics.IncCounter("experimental.uncompressed")
+	}
```

Comment:
We may decide to permanently disable transport compression.
In this case this metric will always be zero and useless.
So I used experimental prefix to hint that we may remove soon.